bears/rest/rstcheckBear.py

```diff
@@ -0,0 +1,50 @@
+from rstcheck import check
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class rstcheckBear(LocalBear):
+    LANGUAGES = {"reStructuredText"}
+    REQUIREMENTS = {PipRequirement('rstcheck')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/ccjtkotaqzgggkkqtgelmj02x'
+
+    def run(self, filename, file, ignore_list: str='', report: str=''):
+        """
+        A Bear for rstcheck (https://pypi.python.org/pypi/rstcheck)
+
+        rstcheck checks syntax of reStructuredText and code blocks
+        nested within it
+        supports bash, c, cpp, json, python and rst
+
+        :param ignore_list: comma seperated list for code blocks to be
```

Comment:
This param name is quite ambiguous. Use a more descriptive name, like `code_block_language_ignore`.